pom.xml

```diff
@@ -128,7 +128,7 @@
     <slf4j.version>1.7.10</slf4j.version>
     <log4j.version>1.2.17</log4j.version>
     <hadoop.version>2.2.0</hadoop.version>
-    <protobuf.version>2.5.0</protobuf.version>
+    <protobuf.version>2.6.1</protobuf.version>
```

Comment:
This isn't necessarily OK since it changes protobuf for the whole project, and could conflict with versions used by, say, Hadoop. 2.4 and 2.5 were terribly incompatible, and I am paranoid about 2.5 vs 2.6.

The KCL update itself is probably fine. Is that helpful at all by itself? you'll have to update the deps files to reflect the updated dependencies for tests to pass.